spaces/S000020/README.md

```diff
@@ -14,6 +15,7 @@ Let $X=\omega\cup\{\infty\}=\{0,1,2\dots\}\cup\{\infty\}$.
 Define $U \subseteq X$ to be open if its complement either is finite or includes $\infty$.
 
 This space is the one-point compactification of a countably infinite discrete space.
+Equivalently, it is homeomorphic to $\{0\}\cup\{2^{-n}:n<\omega\}$ as a subspace of {S25}.
```

Comment:
No need for the word "Equivalently", I think.  Also, maybe slightly simpler: replace $2^{-n}$ with $1/n$.

Reply:
I'm scared of division by zero so I prefer to use 2^{-n}.

Reply:
Well, one could have specified $n=1,2,\dots$, but ok.

Reply:
It's a silly hill, and not quite one I'm willing to die on, but thanks for your understanding. 😆